### PR TITLE
[4] help text change for COM_MAILS_CONFIG_FIELD_ATTACHMENT_FOLDER_DESC

### PR DESCRIPTION
**Pull request by PhilETaylor**
### Summary of Changes

help text change, simplify the wording, and make it make sense.

### Actual result BEFORE applying this Pull Request

Screenshot 2021-05-30 at 13 44 26

### Expected result AFTER applying this Pull Request

Screenshot 2021-05-30 at 13 45 00

### Documentation Changes Required

none

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 5b4085f6cf0373ececd71a74e513cb195d21d5e9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34286">issues.joomla.org/tracker/joomla-cms/34286</a>.</sub>

**Comment by rjharishabh** [comment]
I have tested this item :white_check_mark: successfully on 5b4085f6cf0373ececd71a74e513cb195d21d5e9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34286">issues.joomla.org/tracker/joomla-cms/34286</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34286">issues.joomla.org/tracker/joomla-cms/34286</a>.</sub>

**Comment by Quy** [comment]
Thanks!
